collections/_posts/2025-01-30-SCL-human-error.md

```diff
@@ -0,0 +1,146 @@
+3. Generate another test set with 20% synthetic errors on label by randomly picking a wrong class
+4. Measure similarity between true positives, true negatives and false positives
+5. Plot distributions
+
+A result is shown Figure 2.
+
+![Fig. 2 Error distributions](/collections/images/SCL_human_error/fig2_errors_dist.png)
+
+The overlap between true positives and human errors is higher than between true positives and true negatives and higher than between true positives and synthetic errors. This is a clue that human errors are made between images that have **high visual similarities**. It also explain effectiveness of methods to **tackle synthetic noise** by giving more confidence to pairs closely aligned, and that they may be non effective for human errors.
+
+They interpret it in the case of SCL by saying that **false positives will mostly be easy positives**.
+
+
+# New SCL objective: SCL-RHE
+Their new objective must satisfy two conditions :  (P1) ensuring that the latent class of positive samples match the anchor latent class (i.e. drawing true positives) and (P2) deprioritize easy positives.
+
+They derive a new distribution from which the positive will be sampled:
+
+![Equation 6 for positive distribution](/collections/images/SCL_human_error/eq6_pos_dist.png)
+
+which is formed by a coefficient that gives more weight to samples far from the anchor and a probability distribution which is the true positive distribution (unknown). This distribution is then derived using the perspective of Positive-Unlabeled learning and integrated in the loss. The expectations obtained in the loss are then approximated  and samples are drawn using a Monte Carlo importance sampling strategy. Please refer to the article and its supplementary material for more detailed explanation.
+
+Qualitatively, samples will be mostly drawn from the hard positives and sometimes in the negative set.
+
+They also use this method for negative pairing even if the impact is supposed to be less important.
+
+# Experiments
+Three setups are considered: 
+1. from scratch with human-labelling errors
+2. transfer learning with pretrained weights (with human-labelling error)
+3. pre-training with high levels of synthetic errors
+
+They use the official train-test splits, and different architectures (BEiT/ViT-based and ResNet-50).
+
+## 1. Training from scratch
+SCL is used for a pre-training task then a classification head is added to evaluate the classification performances with a frozen network. The datasets considered are CIFAR10, CIFAR100 and ImageNet-1k (error rate 5.85%).
+
+![Table 1 Training from scratch](/collections/images/SCL_human_error/tab1_scratch.png)
+
+The performances are better than classical supervised and classical Supervised CL (see Table 1). There are also better than Sel-CL and TCL, methods that mitigate synthetic errors, maybe because these methods discard some samples that may deteriorate their performances.
```

Comment:
```suggestion
The performances are better than classical supervised and classical Supervised CL (see Table 1). They are also better than Sel-CL and TCL, methods that mitigate synthetic errors, maybe because these methods discard some samples that may deteriorate their performances.
```